src/render/tutorials/graphql-usage/index.html.md

```diff
@@ -1,9 +1,9 @@
----
-title: Using GraphQL with Knot.x
-author: Voycawojka
-description: "In this tutorial we implement a sample GraphQL endpoint that makes use of Knot.x tasks system."
-order: 7
-date: 2019-10-16
-layout: redirect
-target: /tutorials/graphql-usage/2_0/
----
+---
+title: Using GraphQL with Knot.x
```

Comment:
I'd change the slug to:
`/tutorials/graphql-with-knotx`
that should be better for SEO